wpimath/src/main/native/include/frc/kinematics/MecanumDriveOdometry.h

```diff
@@ -28,28 +28,32 @@ class WPILIB_DLLEXPORT MecanumDriveOdometry {
   /**
    * Constructs a MecanumDriveOdometry object.
    *
+   * IF wheelPositions is unspecified,
+   * You NEED to reset your encoders (to zero).
```

Comment:
This comment no longer seems relevant since the wheelPositions are always specified.